Kickstarter-iOS/Features/PledgeView/Controllers/PledgeViewControllerHelpers.swift

```diff
@@ -0,0 +1,49 @@
+import KsApi
+import Library
+import UIKit
+
+struct PledgeViewControllerHelpers {
```

Comment:
This function is literally copy-pasted from the `pledgeViewController` with no changes; no need to review carefully.